cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/upscale/AzureUpscaleService.java

```diff
@@ -111,6 +110,14 @@
         }
     }
 
+    private List<CloudInstance> getNewInstances(List<CloudResource> newInstances) {
```

Comment:
It seems this part was not unit tested. can you please add a unit test for this method?

Reply:
I agree that it would be useful but I'd prefer doing it in a follow-up as I hardly modified the behaviour and it might cover a Linkedin error. Do you agree?